autogen/pom.xml

```diff
@@ -58,7 +58,7 @@
     </dependency>
     <dependency>
       <groupId>com.github.scopt</groupId>
-      <artifactId>scopt_2.10</artifactId>
+      <artifactId>scopt_2.12</artifactId>
```

Comment:
Can we change this to `scopt_${scala.binary.version}` and build HiBench with `-Dscala=2.12` so that we can support all scala versions?

Reply:
Yes, it will be better.